### PR TITLE
the inevitable Revert "Refactors admin verbs from giant ass lists into datums" in case stuff breaks

### PR DESCRIPTION
**Pull request by Mothblocks**
Reverts tgstation/tgstation#72407

### CODE REVIEW

**Comment by Mothblocks** [comment]
We're gonna revert this into a project branch because it was meant to be test merged but wasn't because I had thought I did

File: code/__DEFINES/ai.dm [hunk 1]
```diff
@@ -51,8 +51,6 @@
 #define BB_NEXT_HUNGRY "BB_NEXT_HUNGRY"
 ///what we're going to eat next
 #define BB_FOOD_TARGET "bb_food_target"
-///Path we should use next time we use the JPS movement datum
-#define BB_PATH_TO_USE "BB_path_to_use"
```
Comment (Mothblocks) [inline code/__DEFINES/ai.dm:55]: wut

Comment (LemonInTheDark) [reply code/__DEFINES/ai.dm:55]: hhhhhhhhhhhhhhhhhh

**Comment by Mothblocks** [comment]
(stuff broke)

**Pull request merged**